[2023-08-01 22:47:08,408] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: s3_to_BigQuery.weather_data_to_BQ manual__2023-08-01T22:47:05.904265+00:00 [queued]>
[2023-08-01 22:47:08,416] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: s3_to_BigQuery.weather_data_to_BQ manual__2023-08-01T22:47:05.904265+00:00 [queued]>
[2023-08-01 22:47:08,416] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-01 22:47:08,417] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-08-01 22:47:08,417] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-01 22:47:09,106] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): weather_data_to_BQ> on 2023-08-01 22:47:05.904265+00:00
[2023-08-01 22:47:09,114] {standard_task_runner.py:52} INFO - Started process 228 to run task
[2023-08-01 22:47:09,119] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 's3_to_BigQuery', 'weather_data_to_BQ', 'manual__2023-08-01T22:47:05.904265+00:00', '--job-id', '199', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpxw1vrvhm', '--error-file', '/tmp/tmppyms304j']
[2023-08-01 22:47:09,121] {standard_task_runner.py:80} INFO - Job 199: Subtask weather_data_to_BQ
[2023-08-01 22:47:09,275] {task_command.py:370} INFO - Running <TaskInstance: s3_to_BigQuery.weather_data_to_BQ manual__2023-08-01T22:47:05.904265+00:00 [running]> on host ab2997d53564
[2023-08-01 22:47:09,501] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=s3_to_BigQuery
AIRFLOW_CTX_TASK_ID=weather_data_to_BQ
AIRFLOW_CTX_EXECUTION_DATE=2023-08-01T22:47:05.904265+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-01T22:47:05.904265+00:00
[2023-08-01 22:47:09,503] {logging_mixin.py:115} INFO - 
 heeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeyy
[2023-08-01 22:47:10,069] {logging_mixin.py:115} INFO - 
✅ DataFrame created ! ✅
[2023-08-01 22:47:10,070] {logging_mixin.py:115} INFO - Error: [Errno 21] Is a directory: '/opt/***/pluginslime-project-key.json'
[2023-08-01 22:47:10,074] {python.py:173} INFO - Done. Returned value was: None
[2023-08-01 22:47:10,125] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=s3_to_BigQuery, task_id=weather_data_to_BQ, execution_date=20230801T224705, start_date=20230801T224708, end_date=20230801T224710
[2023-08-01 22:47:10,214] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-01 22:47:10,332] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
